usdt/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 license = "Apache-2.0"
 description = "Dust your Rust with USDT probes"
 repository = "https://github.com/oxidecomputer/usdt.git"
-rust-version = "1.63.0"
+rust-version = "1.75.0"
 
```

Comment:
Commenting here, rather than lines 32-35, because GH review is bad...

We should be able to get rid of the `asm` feature in the child crates (usdt-impl, usdt-macro, usdt-attr-macro), I think?  Leaving it at the top level makes sense, so that consumers are not forced to update their consumption until such time that we cut a version for breaking changes.

Reply:
Oh good point. I wasn't actually sure whether those features were private or not so I left them in place. Sounds like it's just `usdt` that's public, and the rest are private.

Reply:
Yeah, all the crates besides `usdt` are private implementation details, so I agree with Patrick, it'd be good to remove them. When we do get around to cutting a v0.6.0 and dropping the public features, we can close #57.

Reply:
Dropped non-public features, and added a comment.